src/fxa-geodb.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const db = '../db/cities-db.mmdb';
+const db_backup = '../db/cities-db.mmdb-backup';
```

Comment:
all these probably need to be configurable paths, @jbuck for docker purposes, correct me if wrong.